test/engine/update.test.lua

```diff
@@ -160,3 +160,110 @@ t:update({{'+', '[1]', 50}})
 s:update({1}, {{'=', 'field3', {d = 30, e = 31, f = 32}}})
 
 s:drop()
+
+--
+-- gh-3378: allow update absent nullable fields
+
+-- '!'
+s = box.schema.create_space('test')
```

Comment:
Since this is engine test, let's check it for both memtx and vinyl (just pass engine to create_space method)